web/src/engine/websites/Nightow.ts

```diff
@@ -0,0 +1,27 @@
+import { Tags } from '../Tags';
+import icon from './Nightow.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+function InfoExtractor(anchor: HTMLAnchorElement) {
+    const id = '/online/'+anchor.search;
+    const title = anchor.text.replace('[NightowScans]', '').trim();
+    return { id, title };
+}
+
+@Common.MangaCSS(/^https?:\/\/nightow\.net/, 'div.theList h3.title', Common.ElementLabelExtractor(), true)
+@Common.MangasSinglePageCSS('', 'div.selector2 div.options a', InfoExtractor)
+@Common.ChaptersSinglePageCSS('div.theList div.chapter b a', InfoExtractor)
+@Common.PagesSinglePageJS('imageArray.map(image => "http://nightow.net/online/"+ image);', 1000)
+@Common.ImageDirect()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('nightow', `Nightow`, 'http://nightow.net/', Tags.Language.Spanish, Tags.Media.Manga, Tags.Source.Scanlator);
```

Comment:
If there is no other reason i suggest to remove the trailing `/` slash from the URL, so it is the same for all connectors and developer can assume that `this.URI` never ends with a slash.

Reply:
Oops, typo. I usually make sure trailing / are removed.